src/js/ctrl/demonstrations.ts

```diff
@@ -1211,7 +1211,7 @@ export function getScaleInfoForGroup(state: State, groupId: number): ScaleInfo {
   };
 }
 
-export function cleanSpecForPreview(sceneSpec, width: number, height: number, groupName: string, interactionId: number): Spec {
+export function cleanSpecForPreview(sceneSpec, width: number, height: number, groupName: string, interactionId: number, isHistoryItem: boolean): Spec {
```

Comment:
i think you should actually have a separate function for cleaning history instead of overloading this function in `demonstrations`. it will make more conceptual sense

Reply:
would this be in another file I create in src/js/ctrl?

Reply:
i think it can probably just go in one of your components for now. i moved the interactions one to ctrl so that i could share it between interactions and widgets